scripts/disable-resource-locking.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+SUBS=$1
+
+echo "retrieve all resource groups in a subscription"
+RESOURCE_GROUPS=$(az group list | jq '.[].name')
```

Comment:
Do not put lock on all the resources. Just put locks first on Database/Keyvault/app insights